index.js

```diff
@@ -175,6 +176,7 @@ exports.login = async function (options) {
 };
 
 exports.viewAccount = async function (options) {
+    await eventtracking.askForOptInAndAccountID(options);
```

Comment:
this can be used in scripts, not necessarily user-interactive

Reply:
@vgrichina what do you mean using in scripts?

Reply:
I mean that I can write scripts relying on `near-cli` (like e.g. we have scripts in projects created by create-near-app to use `near deploy`, etc).

E.g. I can try to get formatted amount from this call:

```
[I] ➜ near state account.testnet | grep formattedAmount
  formattedAmount: '328.96507017987727742332'
```

this kind of usage breaks if you have a prompt to user. 

We have few commands where user input is expected in any case (like `near login)` – so these can be used to ask user. Commands which don't necessary require user interaction shouldn't be interfered with. Somebody might use them in CI environment where there is no user to prompt. `near deploy` is quite likely to be used in CI for example.

Reply:
@icerove like e.g. we can see this problem in action on near-cli CI:
https://travis-ci.com/github/near/near-cli/jobs/457657341

it basically hangs on `create-account` call

Reply:
the function `askForConsentIfNeeded` is asking questions to user instead of let user typing in some account ID or give consent from their side. If I understand correctly, writing scripts is not the case we want for asking process. @vgrichina